src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss

```diff
@@ -6,7 +6,7 @@
         height: 100%;
         border-radius: 0;
         margin-top: 0;
-        padding: 16px 0;
+        padding: 0;
```

Comment:
if there is no default padding then remove this padding instead assigning 0

Reply:
a default paading was defined so to counter it.

Reply:
done